src/typeAcquisition.ts

```diff
@@ -569,7 +573,11 @@ const getDependenciesForModule = (
           throw `No outer module or path for a relative import: ${moduleToDownload}`;
         }
 
-        const absolutePathForModule = mapRelativePath(moduleToDownload, path);
+        let absolutePathForModule = mapRelativePath(moduleToDownload, path);
+
+        if (moduleToDownload === '.') {
+          absolutePathForModule += '/index';
+        }
```

Comment:
```suggestion
        // patch the broken logic from the original `typeAcquisition`
        // TODO: replace the whole thing with `@typescript/ata` that has been published since our "fork" was created
        if (moduleToDownload === '.') {
          absolutePathForModule += '/index';
        }
```